tests/unit/test_tag_chartbeat.py

```diff
@@ -95,6 +95,5 @@ def test_render_internal_ip(self):
         req.META['REMOTE_ADDR'] = '1.1.1.1'
         context = Context({'request': req})
         r = ChartbeatBottomNode().render(context)
-        self.assertTrue(r.startswith(
-                '<!-- Chartbeat disabled on internal IP address'), r)
-        self.assertTrue(r.endswith('-->'), r)
+        assert r.startswith('<!-- Chartbeat disabled on internal IP address')
+        assert r.endswith('-->')
```

Comment:
Hey, you win! :trophy: That code is lovely to read. :love_you_gesture: See how it improved. Nice!

Reply:
Because it reads better. It's more concise. It's less code that does the same. This is more maintainable. An investment into a sustainable future of the project.